cmd/kubelet/app/options/options.go

```diff
@@ -131,6 +131,7 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.TLSPrivateKeyFile, "tls-private-key-file", s.TLSPrivateKeyFile, "File containing x509 private key matching --tls-cert-file.")
 	fs.StringVar(&s.CertDirectory, "cert-dir", s.CertDirectory, "The directory where the TLS certs are located (by default /var/run/kubernetes). "+
 		"If --tls-cert-file and --tls-private-key-file are provided, this flag will be ignored.")
+	fs.Int32Var(&s.CertRotation, "experimental-cert-rotation", s.CertRotation, "<Warning: Experimental feature> The percentage threshold of certificate duration remaining before a new key should be generated and signed using the certificate signing request API on the api server. New cert/key pairs generated by this process will be stored in the directory specified by --cert-dir. If less than 1, rotation is disabled. [default=0]")
```

Comment:
I think so far they're just binary flags, and let you tag features as alpha/beta. My suggestion above was that we hardcode a value for this, unless you think it's an important knob to provide.